pkg/schema/load/testdata/malformed_cue/cue/data/gen.cue

```diff
@@ -0,0 +1,214 @@
+package grafanaschema
```

Comment:
do we need the whole file? can we just put the smallest possible malformed CUE file here?